src/App.js

```diff
@@ -13,27 +20,65 @@ const onSelectedGenre = (genre) => {
 }
 
 const genres = [
-  'ALL',
-  'DOCUMENTARY',
-  'COMEDY',
-  'HORROR',
-  'CRIME',
-  'SUSPENSE'
+  'Action', 'Adventure', 'Science', 'Comedy', 'Drama'
```

Comment:
nit: Sounds like genres should come from db, for the scope of the task it could be an array of Genre objects like:
```
const genres = [
  {
    id: 'action',
    value: 'Action'
  }
];
```

Reply:
Fixed